bigtable/metrics_test.go

```diff
@@ -201,9 +202,8 @@ func TestNewBuiltinMetricsTracerFactory(t *testing.T) {
 				t.Errorf("builtinEnabled: got: %v, want: %v", gotClient.metricsTracerFactory.enabled, test.wantBuiltinEnabled)
 			}
 
-			if diff := testutil.Diff(gotClient.metricsTracerFactory.clientAttributes, wantClientAttributes,
-				cmpopts.IgnoreUnexported(attribute.KeyValue{}, attribute.Value{})); diff != "" {
```

Comment:
Removed this as all the fields in attribute.Value are unexported and nothing was getting compared

https://github.com/open-telemetry/opentelemetry-go/blob/v1.29.0/attribute/value.go#L21-L27